specificatie/partner.yaml

```diff
@@ -131,3 +133,23 @@ components:
       properties:
         datum:
           $ref: 'datum.yaml#/components/schemas/GbaDatum'
+    OntbindingHuwelijkPartnerschap:
```

Comment:
Is het niet handig dit als OntbindingHuwelijkPartnerschapBasis te definiëren? Dan kan hierop worden uitgebreid in de historie API.

Reply:
Aangepast